src/tests/test_configobj.py

```diff
@@ -1375,3 +1375,28 @@ def test_writing_out_dict_value_with_unrepr(self):
         c = ConfigObj(cfg, unrepr=True)
         assert repr(c) == "ConfigObj({'thing': {'a': 1}})"
         assert c.write() == ["thing = {'a': 1}"]
+
+
+def test_dividers_happy(cfg_contents):
+    cfg = cfg_contents("""
+[section]
+    ook : 'eek'
+    monkey = True
+    beast + 666
+    """)
+    c = ConfigObj(cfg, dividers=':=+')
+    assert c['section'] is not None
+    assert c['section']['ook'] == 'eek'
+    assert c['section']['monkey'] == 'True'
+    assert c['section']['beast'] == '666'
+
+
+def test_dividers_unhappy(cfg_contents):
+    cfg = cfg_contents(u"""
+[section]
+    ook … 'eek'
+    """)
+    try:
+        c = ConfigObj(cfg, dividers=u'…')
+    except AttributeError as err:
+        assert str(err) == "No valide characters found for dividers."
```

Comment:
The *first* separator will be the one taken. Any other will just be part of the option. So, for example:

`Fred := some weird string`

Will yeild:

`assert c['section']['Fred'] == '= some weird string'`

Is this acceptable?

And yes, this should be documented too! ^_~